web/modules/custom/rdf_etl/src/EtlState.php

```diff
@@ -0,0 +1,55 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\rdf_etl;
+
+/**
+ * Class representing a state of a pipeline.
+ *
+ * @package Drupal\rdf_etl
```

Comment:
I see the `@package` tag used everywhere in this PR but Drupal Coding Standards, the _API documentation and comment standards_ doesn't support this tag https://www.drupal.org/docs/develop/coding-standards/api-documentation-and-comment-standards